tap_exacttarget/state.py

```diff
@@ -28,7 +28,7 @@ def get_last_record_value_for_table(state, table):
         return None
 
     date_obj = datetime.datetime.strptime(raw, DATE_FORMAT)
-    date_obj = date_obj - datetime.timedelta(days=1)
+    date_obj = date_obj - datetime.timedelta(hours=6)
```

Comment:
Since this is separate from the click URL change, can we split this into its own PR?

Reply:
I've created PR #21 for this.

Reply:
This change has been reverted on this branch.